### PR TITLE
Proposed Fix for JuliaLang#42774 

### PR DESCRIPTION
**Pull request by chflood**
using Test
Before:
```
julia> @test :(1 := 2) |> repr == ":($(Expr(:(:=), 1, 2)))"
Test Passed
Expression: :($(Expr(:(:=), 1, 2))) |> repr == ":($(Expr(:(:=), 1, 2)))"
Evaluated: ":(\$(Expr(:(:=), 1, 2)))" == ":(\$(Expr(:(:=), 1, 2)))"
julia>
```
After:
```
julia> @test :(1 := 2) |> repr == ":($(Expr(:(:=), 1, 2)))"
Test Passed
Expression: :(1 := 2) |> repr == ":($(Expr(:(:=), 1, 2)))"
Evaluated: ":(1 := 2)" == ":(1 := 2)"
```

### CODE REVIEW

**Comment by simeonschaub** [comment]
Could you add that test to `test/show.jl`?

File: base/show.jl [hunk 1]
```diff
@@ -1316,7 +1316,7 @@ const uni_ops = Set{Symbol}([:(+), :(-), :(!), :(¬), :(~), :(<:), :(>:), :(√)
 const expr_infix_wide = Set{Symbol}([
     :(=), :(+=), :(-=), :(*=), :(/=), :(\=), :(^=), :(&=), :(|=), :(÷=), :(%=), :(>>>=), :(>>=), :(<<=),
     :(.=), :(.+=), :(.-=), :(.*=), :(./=), :(.\=), :(.^=), :(.&=), :(.|=), :(.÷=), :(.%=), :(.>>>=), :(.>>=), :(.<<=),
-    :(&&), :(||), :(<:), :($=), :(⊻=), :(>:), :(-->)])
+    :(&&), :(||), :(<:), :($=), :(⊻=), :(>:), :(-->), :(:=), :($=)])
```
Comment (simeonschaub) [inline base/show.jl:1319]: `$=` is already listed above.

**Comment by Keno** [comment]
Windows ci failure is #43124, i.e. unrelated and known.

**Comment by Keno** [comment]
Merged, few smaller notes: For commit message titles, we generally prefer the imperative mood. `i.e. Fix #42274`, rather than `i.e. Fix for #42274`. Even better if the commit message says what the actual change is, since that makes it easier to browse the changelog without needing to look up issue numbers, e.g. in this case it could have been `Fix expr printing for ±, ∓, :=, ≔, ⩴, ≕`. With the reference to the issue number in the commit body to make github auto-linking work. Also for code in commit messages, the convention is to markdown-quote it (i.e. wrap it in triple-backquote), since that renders better on GitHub. @fredrikekre fixed that up in the original PR description and I fixed it up in the commit message on merge. Nice work, congrats on your first merged julia PR!

**Pull request merged**